test/recipes/81-test_cmp_cli.t

```diff
@@ -289,7 +294,7 @@ sub load_tests {
 }
 
 sub mock_server_pid {
-    return `lsof -iTCP:$server_port -sTCP:LISTEN | tail -n 1 | awk '{ print \$2 }'`;
+    return `lsof -iTCP:$server_port | grep LISTEN | awk '{ print \$2 }'`;
```

Comment:
What about dropping the `grep` and using`| awk '/LISTEN/ { print \$2 }'` instead?

Reply:
Good thought that we can also avoid using `grep`.
I've just added a check for the existence of `awk`.

Reply:
You don't need to check if AWK is present.  Really?  Perl there but awk isn't? :)
You could also do this all inline in perl code pretty easily.